app/assets/javascripts/initializers/initNotifications.js

```diff
@@ -88,51 +86,55 @@ function fetchNotificationsCount() {
   }
 }
 
+function onClickReaction(event) {
+  event.preventDefault();
+  sendHapticMessage('medium');
+  var thisButt = this;
+  thisButt.classList.add('reacted');
```

Comment:
this is going to fail, this is `undefined` here. on line 125 you pass an arrow function that calls `onClickReaction`, why not just pass `onClickReaction` ?

Remember, arrow functions behave differently when `this` is concerned

ps. what about a name `onClickReactionButton` ?

Reply:
Yeah, thanks for the warning. I've replaced `this` to `event.target`